src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -196,26 +228,25 @@ function createCampaignToken(_, playerColor, _)
     return
   end
 
-  local campaignLog = findUniqueObjectWithTag("CampaignLog")
+  campaignLog = findUniqueObjectWithTag("CampaignLog")
   if campaignLog == nil then
     broadcastToAll("Campaign log not found!", Color.Red)
     return
   end
 
-  local traumaValues = {
-    0, 0, 0, 0,
-    0, 0, 0, 0
-  }
-  local counterData = campaignLog.getVar("ref_buttonData")
-  if counterData ~= nil then
+  additionalIndex = findUniqueObjectWithTag("AllCardsHotfix")
```

Comment:
There can technically be multiple objects with this tag (keep in mind the original usecase for this tag was correcting / adding cards).